### ISSUE TITLE
bad underscores in imported data 

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-studio#1160]

**Comment by ChristopherChudzicki** [comment]
@pdpinch I didn't actually test this, but I suspect that "open in studio, edit, save" won't work if the italicizing underscores have been escaped. Those two pages have escaped italicizing underscores, so I think we need to remove the escapes in admin panel, then save. Or just change to asterisks in admin panel, too.

**Comment by pdpinch** [comment]
The three pages with `needs_admin_fix` set to `TRUE` are all in 16.90. I would fix these myself, but we should probably re-import 16.90 first, after https://github.com/mitodl/ocw-to-hugo/issues/493 is deployed.

**Comment by ChristopherChudzicki** [comment]
@pdpinch Are you re-importing all of it?

If there are any courses that are being entirely re-imported, it might be worth changing ocw-to-hugo to use `*` instead of `_` for italicizing. It's a simple [Turndown config](https://github.com/mixmark-io/turndown#options) (`emDelimiter`).

**Comment by Wassaf-Shahzad** [comment]
> @pdpinch Are you re-importing all of it?
> 
> If so / if there are any courses that are being entirely re-imported, it might be worth changing ocw-to-hugo to use `*` instead of `_` for italicizing. It's a simple [Turndown config](https://github.com/mixmark-io/turndown#options) (`emDelimiter`).

Already done https://github.com/mitodl/ocw-studio/pull/1147 in this PR

**Comment by pdpinch** [comment]
@ChristopherChudzicki do you think we can close this?

**Comment by ChristopherChudzicki** [comment]
@pdpinch I do not think it is resolved, but it is better (520 instances before; now 233).  I added a new sheet with the updated info: https://docs.google.com/spreadsheets/d/1YR9TjoknAM_lP5zVfI6Wv25ynQ0Q6cij8TqxOoU0tDk/edit?usp=sharing

I had forgotten about this issue, but according to my previous comment we should be able to fix these by opening them in studio and saving them without edits (since studio normalizes underscore-italics to asterisk italics).

**Comment by ChristopherChudzicki** [comment]
@pdpinch I've updated the sheet, which also has the query.

Again, the resolution is "replace  `_` with `*` for italics", which can be done by opening the page in Studio, saving, and re-publishing. *I retested a few on RC, and the results look good.*

**Comment by pdpinch** [comment]
Since this is self-healing, I'll ask the OCW team if they want to do it.
